backend/src/zango/apps/dynamic_models/mixin.py

```diff
@@ -0,0 +1,64 @@
+from django.core.exceptions import ValidationError
+
+from zango.apps.appauth.models import AppUserModel, UserRoleModel
+
+
```

Comment:
There are two cases we need to think of:
- one app object having multiple users mapped to it (Eg. Multiple users working under one role at a single Pharmacy)
- multiple app objects linked to the same user with different roles (E.g. the same user manning two different roles in an institution) 
Are we good with both the above requirements?

Reply:
Yes @kc-diabeat both the requirements are possible.
The only consideration is that a single user can be mapped to only one app object per user role